bezier-rs/lib/src/lib.rs

```diff
@@ -177,11 +177,28 @@ impl Bezier {
 				format!("C {} {}, {} {},", handle_start.x, handle_start.y, handle_end.x, handle_end.y)
```

Comment:
Remove the space after the letters and commas, they aren't needed and increase the SVG length.